docs/docsite/rst/dev_guide/platforms/vmware_guidelines.rst

```diff
@@ -0,0 +1,76 @@
+.. _VMware_module_development:
+
+***********************************************
+Guidelines for VMware module module development
+***********************************************
+
+The VWware modules and these guidelines are maintained by the VMware Working Group.  For
+further information the team community page: `https://github.com/ansible/community/wiki/VMware`.
+
+.. contents::
+   :local:
+
+Testing
+=======
+
+Most of the existing modules are covered by functional tests. The tests are located in the :file:`test/integration/targets/`.
+
+Test environment
+----------------
+
+By default, the tests are run against a vcenter API simulator called `vcsim <https://github.com/vmware/govmomi/tree/master/vcsim>`. ``ansible-test`` will automatically pull a vcsim container and use it to set-up the test environment.
+
+You can trigger the test of a module manually with the ``ansible-test`` command. For example, to trigger ``vcenter_folder`` tests:
+
+.. code-block:: shell
+
+    source hacking/env-setup
+    ansible-test integration --python 3.7 vcenter_folder
+
+You will need to install a couple of Python dependencies, like pyVim, pyvmomi and requests.
+
+``vcsim`` is handy because it's much more fast that than a regular test environment. However, it does not
```

Comment:
```suggestion
``govcsim`` is handy because it's much more fast that than a regular test environment. However, it does not
```

Reply:
Applied

Reply:
I'm confused by the difference between `govcsim` and `vcsim` - the project name and repo both use `vcsim`, though, so that's what I think we should use here.

Reply:
I personaly use `vcsim` too, I believe it's clear enough.